database/database_test.go

```diff
@@ -82,15 +84,20 @@ func TestDatabase(t *testing.T) {
 	}
 
 	for testName, test := range tests {
+		// Set test logger to stdout.
+		backend := slog.NewBackend(os.Stdout)
+		log := backend.Logger("test")
+		log.SetLevel(slog.LevelTrace)
```

Comment:
Consider turning this into a test helper function (`newTestLogger` for example) to reuse. I'm also unsure why the logger is being created in a loop here. Consider creating it outside the loop.

Reply:
Good points, will implement both. Thanks

Reply:
The new func can't actually be reused without duplicating it because it is required in two separate packages. I am adding it anyway because its nice to have a func with a self explanatory name rather than a comment.